enodi/PostIt-RoR-
# #156957294 Enable a user add other users to group

#### What does this PR do?
This PR handles adding users to a group

#### Description of Task to be completed?
-  Validate input params.
- Allow users to send a post request to add new users to a group.

#### How should this be manually tested?
- Pull this branch and run `rails s`
- To make a request to the API, use an HTTP client testing tool e.g `postman`.
- To test the group-members route on `postman`, send a post request with valid credentials i.e `user_id` to the URL: `api/v1/group/:group_id/user`.
- The Authorization token should be placed in the `headers`.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[Story ID: 156957294](https://www.pivotaltracker.com/n/projects/2166405/stories/156957294)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29783644/39073756-ca51c5a4-44e6-11e8-8038-b9e4b3c02cf1.png)
